chapter3.adoc

```diff
@@ -54,6 +54,17 @@ A dedicated error code, security fault error (cmderr 6), is included in `cmderr`
 
 The error raised by resethaltreq, reset can be identified through the fields `allsecfault` and `anysecfault` in dmstatus. Error status bits are internally maintained for each hart, with the `allsecfault` and `anysecfault` fields indicating the error status of the currently selected harts. These error statuses are sticky and can only be cleared by writing 1 to `acksecfault` in dmcs2.
 
+=== Non-secure Debug control
+
+The state element `nsecdbg` is introduced to retain full debugging capabilities, as if the extensions in this specification were not implemented. When `nsecdbg` is set to 1:
+
+* The Debug Module gains M-mode debug privilege for all harts in the system.
```

Comment:
This sentence is not clear. Maybe you can replace it with something like this one: All debug operations are executed at machine mode privilege level.

Reply:
Thank you. The revised version is clear.